Tasks/Chernousik/Task1/Task1.py

```diff
@@ -0,0 +1,18 @@
+print("Hello, thank you very much for choosing to use the services of our online banking.")
+a = int(input('Enter the value of the deposit \n'))
+b = int(input('Enter time period(years) \n'))
+c = float(input('Enter rate \n'))                        
```

Comment:
the decimal data type would be more suitable for money-related calculations. -1 point for the unwise choice